R/animint.R

```diff
@@ -856,7 +858,9 @@ animint2dir <- function(plot.list, out.dir = tempfile(),
         update.vars <- L$mapping[is.ss | is.cs]
         has.var <- update.vars %in% names(L$data)
         if(!all(has.var)){
-          print(update.vars[!has.var])
+          print(L)
+          print(list(problem.aes=update.vars[!has.var],
+                     data.variables=names(L$data)))
           stop("data does not have interactive variables")
```

Comment:
I like to avoid `print()` statements since they are hard to suppress, but I think I'm OK with this (since it's essentially part of an error message).